client/app/scripts/liveblog-edit/controllers/blog-edit.js

```diff
@@ -174,6 +194,18 @@ define([
             selectPostType: function(postType) {
                 $scope.selectedPostType = postType;
                 $scope.toggleTypePostDialog();
+                if (isPostScorecard()) {
+                    blogService.get($scope.blog._id).then(function(currentBlog) {
```

Comment:
We keep the last scorecard (with the 'remember scores' status) in the blog preferences so we need to load that. It is also needed in case somebody else saved the last scorecard( and the remember scorers status of course)